lib/maven/parse/fromPom.ts

```diff
@@ -58,6 +82,29 @@ async function findDeclaredDependenciesWith(p: Project,
     return { dependencies };
 }
 
+/**
+ * Find declared plugins using the given path expression.
+ * Control over the path expression allows us to look under pluginManagement,
+ * or directly in plugin section under project/build
+ * @param {Project} p
+ * @param {string} pathExpression
+ * @param {string} glob
+ * @param {FunctionRegistry} functionRegistry
+ * @return {Promise<Dependencies>}
+ */
+async function findDeclaredPluginsWith(p: Project,
```

Comment:
this is cool, I like that it wraps applying the path expression and then translating the results into something useful